var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -41,8 +41,9 @@ class Legion(CMakePackage):
     depends_on('mpi', when='conduit=mpi')
     depends_on('cuda@10.0:11.9', when='+cuda_unsupported_compiler')
     depends_on('cuda@10.0:11.9', when='+cuda')
+    depends_on('hip@4.1.0:', when='+hip')
     depends_on('hdf5', when='+hdf5')
-    depends_on('hwloc', when='+hwloc')
+    depends_on('hwloc@1.11.13', when='+hwloc')
```

Comment:
Similar question here, if you can support a range of versions of hwloc, that is preferred, so that the user doesn't have to recompile if they have an older/newer hwloc installed already.

Reply:
I will check -- we are definitely < 2.x right now but I don't know how far back we can go but I would suspect 1.11.X would work. This is a very lightly used feature set so most will not enable it but I switch to 1.11: would probably be enough.  thanks again for the review.

Reply:
pushed an update to add a version range for hwloc.